aiohttp/abc.py

```diff
@@ -230,6 +230,13 @@ async def write_headers(
     ) -> None:
         """Write HTTP headers"""
 
+    def send_headers(self) -> None:
```

Comment:
We don't document `AbstractStreamWriter` but its here so some technically could be using it